crates/configuration/src/utils.rs

```diff
@@ -1,3 +1,7 @@
 pub fn is_true(value: &bool) -> bool {
     *value
 }
+
+pub fn default_as_true() -> bool {
+    true
+}
```

Comment:
Asking myself if those functions can be made ```pub(crate)``` ?

Reply:
yes they can 👍🏽